usr/share/device-quirks/scripts/asus/scripts.d/ally_pipewire.sh

```diff
@@ -1,28 +1,34 @@
 #!/bin/bash
 
-# Returning a unique id to allow users to toggle on or off in /etc/device-quirks/device-quirks.conf
```

Comment:
Please retain the comments, they are useful when we look at this again in a year or so.